onion_store.go

```diff
@@ -2,7 +2,6 @@ package aperture
 
 import (
 	"context"
```

Comment:
nit: commit title should ideally be no more than 50 chars and commit messages should be wrapped at 72 since `git log` doesn't do any formatting: https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#model-git-commit-messages